bigquery_etl/cli/backfill.py

```diff
@@ -401,12 +402,27 @@ def initiate(
     )
 
     # deploy backfill staging table
-    ctx.invoke(
-        deploy,
-        name=f"{dataset}.{table}",
-        project_id=project,
-        destination_table=backfill_staging_qualified_table_name,
-    )
+    try:
+        ctx.invoke(
+            deploy,
+            name=f"{project}.{dataset}.{table}",
+            project_id=project,
+            destination_table=backfill_staging_qualified_table_name,
+        )
```

Comment:
I think the suggestion [here](https://github.com/mozilla/bigquery-etl/pull/5646#pullrequestreview-2074163526) is a good one - maybe we add a call to `query schema update` here in the short-term? Medium/long-term like with `query backfill` I think we should break the dependency on click commands.